.github/workflows/tests.yaml

```diff
@@ -45,23 +46,26 @@ jobs:
           phpts: ts
           debug: true
       -
-        uses: actions/checkout@v4
-        name: Checkout watcher
-        with:
-          repository: e-dant/watcher
-          ref: ${{ env.EDANT_WATCHER_VERSION }}
-          path: 'edant/watcher'
+        name: Download e-dant/watcher
+        run: |
+          mkdir watcher
+          gh release download --repo e-dant/watcher -A tar.gz -O - | tar -xz -C watcher --strip-components 1
```

Comment:
Which branch does this currently download? Wouldn't you need the 'next' branch for this to work without the header?

Reply:
This download the archive of the latest tag. This should work now, a new release has been released by Will a few hours ago.